util/MigratorCLI/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Bit.Migrator;
+using CommandDotNet;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Logging;
+
+internal class Program
+{
+    private static int Main(string[] args)
+    {
+        return new AppRunner<Program>().Run(args);
+    }
+
+    [DefaultCommand]
+    public void Execute(
+        [Option('c', Description = "Connection string to the database on which migrations will be executed.")]
+        string databaseConnectionString,
+        [Option('v')]
+        bool verbose) => MigrateDatabase(databaseConnectionString, verbose);
+
+    private static void MigrateDatabase(string databaseConnectionString, bool verbose = false, int attempt = 1)
+    {
+        var logger = CreateLogger(verbose);
```

Comment:
⚠️ The logs below are emitted inside the migrator itself, and we can just provide a logger to that invocation and eliminate the console outputs here vs. directing the logger's output to the console.

Reply:
Thanks! I removed the logs from the console app, and removed the check for `enableLogging` flag on general logs (like starting migrations, and migrations succeeded/failed), to give the user the minimum knowledge of what's going on, and what's the result.